docs/releases/1.10.txt

```diff
@@ -600,6 +600,17 @@ You can check if your database has any of the removed hashers like this::
     # Unsalted MD5 passwords might not have an 'md5$$' prefix:
     User.objects.filter(password__length=32)
 
+``TransactionTestCase`` serialized data loading
+-----------------------------------------------
+
+Initial data migrations are now loaded in ``TransactionTestCase`` at the end of
+the test, after the database flush.
+In older versions, these data were loaded at the beginning of the test, but
+this behavior was preventing ``--keepdb`` option to work properly (the
+database was empty at the end of the whole test suite).
+It shouldn't have any impact on your test suite if you have not customized
```

Comment:
I think that my explanation is partially wrong. It won't have any impact on test suites if people have defined all their `TransactionTestCase` with `serialised_rollback` option to `True`.

If somebody is using `serialized_rollback` in just one test, he is expecting to have the initial data loaded at the beginning of the test, not the end. And it makes hard to predict: by loading the data at the end of the test, we know that we prepare a clean environment for the next test, which is unknown by the developer. It then makes sense only if all the `TransactionTestCase` have the same `serialized_rollback` logic.